app/Main.hs

```diff
@@ -30,21 +44,34 @@ main = execParser opts >>= dotEnv
      <> header "dotenv - loads options from dotenv files" )
 
 config :: Parser Options
-config = Options
-     <$> some (strOption (
-                  long "file"
-                  <> short 'f'
-                  <> metavar "FILE"
-                  <> help "File to read for options" ))
+config =
+  Options
+     <$> argument str (metavar "PROGRAM")
+
+     <*> strOption
+          ( long "env"
+          <> value ".env"
+          <> metavar "\"DOTENV FILE\""
```

Comment:
We could use `showDefault` here. Also: I don't understand why you need to quote manually contents of `metavar`. Maybe just `"DOTENV_FILE"` will do?